app/src/main/res/layout/view_bookmark_entry.xml

```diff
@@ -22,7 +22,7 @@
     android:paddingTop="14dp"
     android:paddingBottom="14dp"
     android:paddingStart="16dp"
-    android:paddingEnd="16dp"
+    android:paddingEnd="10dp"
```

Comment:
change suggested in the Product Review to align the overflow menu of the toolbar with the overflow menu of each bookmark item